wherehows-frontend/conf/application.conf

```diff
@@ -61,10 +61,11 @@ database.opensource.password = ${WHZ_DB_PASSWORD}
 database.opensource.url = ${WHZ_DB_URL}
 
 search.engine = "default"
-search.engine = ${?YOUR_SEARCH_ENGINE}
+search.engine = ${?WHZ_SEARCH_ENGINE}
 
-elasticsearch.dataset.url = "$YOUR_DATASET_INDEX_URL"
-elasticsearch.flow.url = "$YOUR_FLOW_INDEX_URL"
+elasticsearch.dataset.url = ${WHZ_ES_DATASET_URL}
```

Comment:
For these directive changes, application.env file will be added for the frontend after this PR merge.